cfgov/prepaid_agreements/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+
+
+class PrepaidProduct(models.Model):
+    name = models.CharField(blank=True, max_length=255)
+    issuer_name = models.CharField(max_length=255, blank=True)
+    prepaid_type = models.CharField(max_length=255, blank=True, null=True)
+    program_manager = models.CharField(max_length=255, blank=True, null=True)
+    program_manager_exists = models.CharField(max_length=255, blank=True, null=True)
+    other_relevant_parties = models.TextField(blank=True, null=True)
+    status = models.TextField(blank=True, null=True)
+    withdrawal_date = models.DateField(blank=True, null=True)
+
+    def __str__(self):
+        return self.name
+
+    @property
+    def latest_agreement(self):
+        return self.prepaidagreement_set.order_by('-pk').first()
```

Comment:
Is the assumption that the data being loaded is always ordered in the input file with the latest agreements last? In other words, this doesn't use the `effective_date` field of the agreement, but the order that it was loaded.

Additionally, is this `order_by` clause needed if `PrepaidAgreement` already has a default ordering in its `class Meta`?

Reply:
Yes, I'll add some docstrings here (and elsewhere more generally) to explain this... but basically the data is a little messy and there exist many agreements with the same `effective_date`, so we decided to use the ID as a proxy for which was uploaded most recently, since on Salesforce's end the IDs simply increment.  We're not generating our own IDs here, so they reflect the order in which they were created in Salesforce.  

Good point about the order_by - it's likely redundant and I can remove it.

Reply:
I've removed the order_by since it's redundant, though slightly worried about readability. 

I've also updated this to be based on the created_date instead of the agreement ID.  Unfortunately, effective_dates are not reliable on their own since issuers sometimes upload duplicates.